charts/deployment-operator/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: deployment-operator
 description: creates a new instance of the plural deployment operator
-appVersion: 0.4.13
-version: 0.4.13
+appVersion: 0.4.14
```

Comment:
I don't believe this tag has been created yet has it?

Reply:
I've changed it. Should I reverse this change?